### ISSUE TITLE
Accessibility issue with input hints

### ISSUE THREAD

**Issue by paulGeoghegan** [shoelace-style/webawesome#1186]
### Describe the bug
Hints on Inputs are not read out by screen readers.

### To Reproduce

1. Go to [input docs page](https://webawesome.com/docs/components/input/)
2. Turn on a screen reader e.g. NVDA
3. Scroll down to the hint example and click on the input
4. The hint is not read to the screen reader

### Browser / OS
 - OS: Windows
 - Browser: Firefox
 - Browser version: 140.0.4

### Additional information
You can probably just use aria-describedby to link the input to the hint text.

**Comment by paulGeoghegan** [comment]
I checked if this issue also affected checkboxes but they seem to read their hints correctly.

**Comment by claviska** [comment]
Thanks for the report. We'll look into this!

**Comment by claviska** [comment]
Thanks for reporting this! It's been fixed in #1192. The ID went missing.

**Comment by paulGeoghegan** [comment]
I assumed something like that happened because I thought it used to work.
Thanks for the quick fix!
